actions/test/run_local.ts

```diff
@@ -18,11 +19,12 @@ const main = async () => {
   const network = process.env.NETWORK;
   assert(network, "network is required");
 
-  const testRuntime = await _getRunTime(network);
+  const chainId = toChainId(network);
+  const testRuntime = await _getRunTime(chainId);
 
   // Get provider
-  const provider = await getProvider(testRuntime.context, network);
-  const { chainId } = await provider.getNetwork();
+  const provider = await getProvider(testRuntime.context, chainId);
+  await provider.getNetwork();
```

Comment:
Not storing the result?

Reply:
Nup, we don't need. It's just a quick check to see if its working t. This is for the local run...

I believe it was already there, but i will delete.